var/spack/repos/builtin/packages/parallelio/package.py

```diff
@@ -12,10 +12,11 @@ class Parallelio(CMakePackage):
     large numbers of processors on a HPC system."""
 
     homepage = "https://ncar.github.io/ParallelIO/"
-    url      = "https://github.com/NCAR/ParallelIO/archive/pio2_5_2.tar.gz"
+    url      = "https://github.com/NCAR/ParallelIO/archive/pio2_5_4.tar.gz"
```

Comment:
FYI. This change isn't technically necessary here since Spack uses version substitution.